iset.mm

```diff
@@ -139302,10 +139302,19 @@ is a function if the structure (without the empty set) is a function.
   baseid $p |- Base = Slot ( Base ` ndx ) $=
     ( cbs c1 df-base 1nn ndxid ) ABCDE $.
 
-  $( Index value of the base set extractor.  (Normally it is preferred to work
-     with ` ( Base `` ndx ) ` rather than the hard-coded ` 1 ` in order to make
-     structure theorems portable.  This is an example of how to obtain it when
-     needed.)  (New usage is discouraged.)  (Contributed by Mario Carneiro,
+  $( Index value of the base set extractor.
+
+     Normally it is preferred to work with ` ( Base `` ndx ) ` rather than the
+     hard-coded ` 1 ` .  For example, using theorems such as ~ baseid and
+     ~ basendxnn .
+
+     The main circumstance in which it is necessary to look at indexes directly
+     is when showing that a set of indexes are disjoint, in proofs such as
+     ~ lmodstrd .  Although we have a few theorems such as ~ basendxnplusgndx ,
+     we do not intend to add such theorems for every pair of indexes (which
+     would be quadradically many in the number of indexes).
+
+     (New usage is discouraged.)  (Contributed by Mario Carneiro,
```

Comment:
I prefer comments of the form:
```
$( Blabla.  (Contributed by AA, xx-xxx-xxxx.)  Reason why its usage is discouraged.
   (New usage is discouraged.) $)
```
So here, `(Contributed by Mario Carneiro, 2-Aug-2013.)` would be placed after the first sentence.

Reply:
I guess that works OK where the statement of why it is discouraged is a brief note, but here that's two paragraphs and combined with various discussion of how to use the theorem.

The more common format is generally to keep the `Contributed by` and other parentheticals at the end, so that's what I'd stick with.

Reply:
Not a problem to me, but either way is fine. Resolving.